doc/rst/users/build.rst

```diff
@@ -34,9 +45,34 @@ package manager (https://github.com/spack/spack).
 SCR and all of its dependencies exist in a Spack package. After downloading
 Spack, simply type::
 
-  spack install scr
+  spack compiler find --scope site
+  spack install scr@develop resource_manager=SLURM
+
+This will install the component libraries, DTCMP, LWGRP, libyogrt, and pdsh packages (and possibly an MPI and a
+C compiler if needed).
+
+By default, Spack will attempt to build all dependencies for SCR, including many packages such as MPI and OpenSSL that are already installed on many systems. Often it is preferrable to use the pre-installed versions. Spack uses file called 'packages.yaml' to tell it where to find external packages. Information on setting up 'packages.yaml' is available in the Spack documentation (see https://spack.readthedocs.io/en/latest/configuration.html). Spack searches for the packages.yaml file in several locations. When installing Spack in user space, customizations can be placed in $SPACK/etc/packages.yaml or $HOME/.spack/packages.yaml.
+
+Here is an example of a simple `packages.yaml` file::
+
+    packages:
+      openmpi:
+        buildable: false
```

Comment:
Would someone also need to include a prefix and spec here?

If so, let's add some made up, but realistic values.

Reply:
I don't think we're using openmpi anywhere, so buildable: false just prevents spack from building it. We can either
- put in dummy values
- remove openmpi as a possible provider for mpi, leaving mvapich and spectrum-mpi in packages.yaml.
- Leave it as-is, which I think implies that the compiler finds it without any flags/info.
- use the module method (quartz does have an openmpi module) to declare it

```
 openmpi:
    buildable: false
    externals:
    - spec: openmpi@4.1.0
      modules:
      - openmpi/4.1.0
```

Reply:
Added some bogus information, as well as the real mvapich2 and spectrum-mpi lines from my packages.yaml

Reply:
Ok, thanks.  We can chat about the options on the call.

I do like listing openmpi as an example, since that's a common MPI library.